js/products/displayProducts.js

```diff
@@ -0,0 +1,101 @@
+import { deleteItemData, editItemData } from "../fetch.js";
+
+const container = document.querySelector(".container");
+
+//edit item
+const editButton = document.querySelector("#edit");
+const modalEditItem = document.querySelector("#editItemModal");
+const editItem = document.querySelector(".edit-product");
+const inputEditTitle = document.querySelector("#editTitle");
+const inputEditDescription = document.querySelector("#editDescription");
+const inputEditPrice = document.querySelector("#editPrice");
+const inputEditImageUrl = document.querySelector("#editImageUrl");
+const close = document.querySelectorAll(".closeEditModal")[0];
+
+const displayProducts = (productData) => {
+  const products = productData
+    .map(
+      (product) => `
+      <div class="card-container">
+        <h3 class="card-price">$${product.price}</h3>
+        <h2 class="card-title">${product.title}</h2>
+        <figure class="card-image-container">
+          <img src='${product.imageUrl}' alt="product-image" />
+        </figure>
+        <div class="card-buttons">
+          <button class="btn btn--edit" id="edit-${product.id}" type="button">Edit</button>
+          <button class="btn btn--delete" id="delete-${product.id}" type="button">Delete</button>
+        </div>
+      </div>
+    `
+    )
+    .join("");
+
+  container.innerHTML += products;
+
+  productData.map((product) => {
```

Comment:
Consider using `forEach` instead of `map` as you are producing side effects in the function instead of mapping data.